src/shared/ui/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import "./SearchForm.css";
+
+const SearchForm = ({ initialValue, onSearch, placeholder }) => {
+  const [queryValue, setQueryValue] = useState(initialValue);
```

Comment:
I believe it's more like: 
```suggestion
  const [searchValue, setSearchValue] = useState(initialValue);
```

Reply:
done